### PR TITLE
[MIG] sale_order_line_description: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
Standard migration
Add oca fragments for readme information

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
It has the same UI problem than purchase one.

Please also squash commit history by author.

**Comment by cubells** [comment]
@pedrobaeza done!

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
@cubells I have marked the check, but the code stills appears in the quotation:
seleccion_023

**Comment by cubells** [comment]
@pedrobaeza you must activate two checks: one in **Sales > Settings > Sales & Orders > Product Sale
   description** and once you've checked this general option, you must check the option "Use only product purchase description on order lines" for every user you want.

I've updated readme to make it clearer.

File: sale_order_line_description/README.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+#. Go to **Sales > Settings > Quotations & Orders** and check "Product Sale
+   description* to use only product sale description on the sales order lines.
+#. Go to **Setting > Users & Companies > Users** and check you have checked
+   the "Use only product purchase description on order lines" access right for
+   your user.
+#. Add or modify a sale description to any of your products.
+#. Create a sale order and add a sale order line with this product and check
+   out that now sale order line description only contains the description you
+   are created.
```
Comment (tafaRU) [inline sale_order_line_description/README.rst:55]: `you are created`: could you please rephrase it?

**Review by tafaRU — CHANGES_REQUESTED** [review]
Only one request, the rest LGTM.
Thanks!

**Review by lfreeke — APPROVED** [review]
:+1: (functional test)

**Review by nikul-serpentcs — APPROVED** [review]
@cubells The rest LGTM :+1: : Only minor change, Some commits are Transifex, Can you please Fix it.
mig sale_order_line_description migration to 11 0 by cubells pull request 759 oca sale workflow

File: sale_order_line_description/README.rst (hunk 1 above)
Comment (cubells) [reply sale_order_line_description/README.rst:55]: @tafaRU can you recheck now?

**Comment by cubells** [comment]
@nikul-serpentcs done!

File: sale_order_line_description/README.rst (hunk 1 above)
Comment (tafaRU) [reply sale_order_line_description/README.rst:55]: Ok, now the `USAGE` section LGTM. Thanks!

**Review by tafaRU — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**